api/tests/WebsiteMedia/WebsiteMediaCreateDeleteTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\WebsiteMedia;
+
+use App\Entity\WebsiteMedia;
+use App\Repository\MediaObjectRepository;
+use App\Repository\WebsiteMediaRepository;
+use App\Tests\AbstractTestCase;
+use League\Flysystem\FilesystemOperator;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\HttpFoundation\Response;
+
+class WebsiteMediaCreateDeleteTest  extends AbstractTestCase
+{
+    public const IMAGE_NAME = 'ratpi.png';
+
+    public const WEBSITE_MEDIA_ID = 'test_website_media';
+
+    private MediaObjectRepository $mediaObjectRepository;
+
+    private WebsiteMediaRepository $websiteMediaRepository;
+
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->mediaObjectRepository = self::getContainer()->get(MediaObjectRepository::class);
+        $this->websiteMediaRepository = self::getContainer()->get(WebsiteMediaRepository::class);
+    }
+
+    public function testCreateWebsiteMediaByAdmin(): void
+    {
+        // Create a mediaObject for the test with an admin role
+        $file = new UploadedFile(sprintf('%s/../../fixtures/%s', __DIR__, self::IMAGE_NAME), self::IMAGE_NAME);
+        $mediaResponse = $this->createClientAuthAsAdmin()->request('POST', '/media_objects/images', [
+            'headers' => ['Content-Type' => 'multipart/form-data'],
+            'extra' => [
+                'files' => [
+                    'file' => $file,
+                ],
+            ],
+        ]);
+
+        self::assertResponseIsSuccessful();
+
+        //create a website media with the media object and a predefined test id with an admin role
+        $this->createClientAuthAsAdmin()->request('POST', '/website_media', [
+            'headers' => ['Content-Type' => 'application/ld+json'],
+            'json' => [
+                'id' => self::WEBSITE_MEDIA_ID,
+                'media' => $mediaResponse->toArray()['@id'],
+            ]
+        ]);
+
+        self::assertResponseIsSuccessful();
+
+        // Check if created website media exist
+        $websiteMediaCreated = $this->websiteMediaRepository->findOneBy(['id' => self::WEBSITE_MEDIA_ID]);
+        self::assertNotEmpty($websiteMediaCreated);
```

Comment:
`findOneBy()` return an object or null, you can replace `assertNotEmpty ` by `assertNotNull `